setup.py

```diff
@@ -32,6 +32,8 @@
             "responses==0.10.14",
             "google-cloud-translate==2.0.1",
             "paramiko>=2.8.0",
+            "boto3>=1.21.21",
+            "google>=3.0.0",
```

Comment:
Related to the addition to `optional_dependencies.txt`, `boto3-stubs-lite[essential]>=1.21.21` will need to be added here as well.